Makefile

```diff
@@ -65,7 +71,7 @@ clean:
 .PHONY: isclean
 .SILENT: isclean
 isclean:
-	(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." && exit 1)
+	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." && exit 1)
```

Comment:
There's `.SILENT: isclean` which should do the same thing but for the whole target.  Is it not working?

Reply:
Oh! You're right. Maybe I was seeing the one from the operator? Not sure. I'll remove it

Reply:
Operator doesn't have it, you're right.  It does @ I think.. I didn't know about `.SILENT` when I did that :D